Src/library/base/elks/kernel/routine.e

```diff
@@ -405,14 +433,6 @@ feature {NONE} -- Implementation
 			Result := a.generating_type.type_id
 		end
 
-feature {NONE} -- Externals
-
-	eif_gen_typecode_str (obj: POINTER): STRING
```

Comment:
I tried to remove this C function from the runtime, but it is still used in:
eweasel/tests/term036/routine.e
eweasel/tests/valid068/routine.e
historic/library/base_original/kernel/classic/routine.e
Src/experimental/library/base/elks/kernel/routine.e